NickvisionTagger.Shared/Models/MusicFile.cs

```diff
@@ -854,23 +854,31 @@ public override bool Equals(object? obj)
     /// <returns>True if a < b, else false</returns>
     public static bool operator <(MusicFile? a, MusicFile? b)
     {
+        if(SortFilesBy == SortBy.Path)
+        {
+            return a?.Path.CompareTo(b?.Path) == -1;
+        }
         if(SortFilesBy == SortBy.Title)
         {
             return a?.Title.CompareTo(b?.Title) == -1;
         }
-        else if(SortFilesBy == SortBy.Track)
+        if(SortFilesBy == SortBy.Artist)
         {
-            return a?.Track.CompareTo(b?.Track) == -1 || a?.Track.CompareTo(b?.Track) == 0 && a?.Title.CompareTo(b?.Title) == -1;
+            return a?.Artist.CompareTo(b?.Artist) == -1 || a?.Artist.CompareTo(b?.Artist) == 0 && a?.Track.CompareTo(b?.Track) == -1;
```

Comment:
Perhaps `a?.Artist == b?.Artist` is preferable to `a?.Artist.CompareTo(b?.Artist) == 0` as it would be both shorter, easier to understand, and [very very very slightly] more performant [if you had a million songs]

Reply:
Sorry got caught up in the CompareTo world here haha